prototype/frameworks/langgraph/insurance_workflow.py

```diff
@@ -0,0 +1,141 @@
+import random
+from typing import Annotated, TypedDict
+
+from langchain_ollama import ChatOllama
+from langchain_core.messages import AIMessage, HumanMessage
+from langgraph.graph import StateGraph, START, END
+from langgraph.graph.message import add_messages
+
+# Define the state structure
+class State(TypedDict):
+    messages: Annotated[list, add_messages]
+    score: int
+    needs_scoring: bool
+    needs_approval: bool
+
+# Initialize the LLM model (point this to your Ollama instance or other model endpoint)
+model = ChatOllama(model='mistral:latest')
```

Comment:
Could you instead create a `.env` to allow for users to specify their choice of model, model endpoint and model api key? i.e. it could either be a local model running via ollama or a model hosted somewhere else